<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="gameLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  BlackJack</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>721</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="betButton">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bet</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back To Menu</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="hitButton">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="standButton">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Stand</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="chooseBetButton">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>340</y>
     <width>70</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose
Bet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incrementButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>355</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/icons/plus_Icon.png</normaloff>images/icons/plus_Icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="decrementButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>355</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/icons/minus_Icon.png</normaloff>images/icons/minus_Icon.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
